src/Orm/Zed/SelfServicePortal/Persistence/SpySspAssetToSspModel.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * This file is part of the Spryker Commerce OS.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+declare(strict_types = 1);
+
+namespace Orm\Zed\SelfServicePortal\Persistence;
+
+use Orm\Zed\SelfServicePortal\Persistence\Base\SpySspAssetToSspModel as BaseSpySspAssetToSspModel;
```

Comment:
* We have to use the core one `AbstractSpySspAssetToSspModel` as a base class, same for the classes below.